sass/_aesop.scss

```diff
@@ -290,3 +320,43 @@
 #aesop-chapter-menu {
 	display: none; // fix meeeee
 }
+
+.aesop-grid {
+    &-image {
+        width: inherit; // overrides inline style/property
+    }
+    
+    &-gallery {
+        .aesop-grid-gallery-caption {
+            @include rem(font-size, 16px);
+            background-color: transparent;
+            background-image: linear-gradient(rgba(0, 0, 0, 0), rgba(0, 0, 0, .5));
+            bottom: 0;
+            color: #fff;
+            left: 0;
+            padding: 40px 20px 20px;
+            right: 0;
+            top: auto;
+            z-index: 1; // keeps caption above image
+        }
+        
+        &-item {
+            text-align: center;
+        }
+    }
+}
+
+.fotorama {
+    &__caption__wrap {
+        @include rem(font-size, 16px);
+        background-color: transparent;
+        background-image: linear-gradient(rgba(0, 0, 0, 0), rgba(0, 0, 0, .5));
+        color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.